compiler/types/UsageEnv.hs

```diff
@@ -52,7 +69,7 @@ scaleUE w (UsageEnv e) = UsageEnv $
 
 supUE :: UsageEnv -> UsageEnv -> UsageEnv
 supUE (UsageEnv e1) (UsageEnv e2) = UsageEnv $
-  plusNameEnv_CD sup e1 Zero e2 Zero
+  plusNameEnv_CD sup e1 Omega e2 Omega
```

Comment:
This will bite us if we want to add an affine multiplicity or something :confused: . I'd rather we found a way to make this function compute on usages.